src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportDecisionMaker.cs

```diff
@@ -138,10 +151,59 @@ private string EvaluateSpec(IRejectWithReason spec, LocalEpisode localEpisode)
                 //e.Data.Add("report", remoteEpisode.Report.ToJson());
                 //e.Data.Add("parsed", remoteEpisode.ParsedEpisodeInfo.ToJson());
                 _logger.ErrorException("Couldn't evaluate decision on " + localEpisode.Path, e);
-                return string.Format("{0}: {1}", spec.GetType().Name, e.Message);
+                return String.Format("{0}: {1}", spec.GetType().Name, e.Message);
             }
 
             return null;
         }
+
+        private bool ParseFolder(List<string> videoFiles, Series series, ParsedEpisodeInfo folderInfo)
```

Comment:
This doesn't parse the folder, If I understand the purpose of it correctly then it should be called `ShouldUseFolderName`?